pkg/bpf/pinning.go

```diff
@@ -56,7 +56,7 @@ func incompatibleMaps(spec *ebpf.CollectionSpec, opts ebpf.CollectionOptions) ([
 
 		pinPath := path.Join(opts.Maps.PinPath, ms.Name)
 		m, err := ebpf.LoadPinnedMap(pinPath, nil)
-		if errors.Is(err, unix.ENOENT) {
+		if errors.Is(err, fs.ErrNotExist) {
```

Comment:
Don't change the error. Move the function in a Linux specific file instead.

Reply:
Changed the file name to pinning_linux.go since the whole file will not work on windows